crates/sui-indexer-alt-jsonrpc/src/data/kv_loader.rs

```diff
@@ -26,6 +34,12 @@ pub(crate) enum KvLoader {
     Pg(Arc<DataLoader<PgReader>>),
 }
 
+/// A wrapper for the contents of a transaction, either from Bigtable or Postgres.
```

Comment:
Maybe worth adding a little explanation of why we need this enum (to avoid eagerly deserializing stuff). Also thoughts on this type being elsewhere, like `data/transactions.rs`? (I'm in two minds).